src/core/io/src/4C_io_visualization_parameters.hpp

```diff
@@ -45,6 +46,10 @@ namespace Core::IO
     //! Enum containing the type of output data format, i.e., binary or ascii.
     OutputDataFormat data_format_;
 
+    //! Enum containing the compression level used when writing output (no_compression,
+    //! best_compression, best_speed). Default: best_speed.
+    LibB64::CompressionLevel compression_level_;
```

Comment:
You are repeating the values here and also the default. I would move this to the enum itself.